s3-eventbridge-gluecrawler/README.md

```diff
@@ -0,0 +1,60 @@
+# AWS Service 1 to AWS Service 2
```

Comment:
Are these files under the `s3-eventbridge-gluecrawler` folder added in error? Can you please review these?

Reply:
Resolved - sorry it was my mistake